frontend/src/components/common/Notification.js

```diff
@@ -0,0 +1,54 @@
+import { alert, Stack, defaultModules } from "@pnotify/core";
+import * as PNotifyMobile from "@pnotify/mobile";
+import _ from "lodash-es";
+
+// Set up PNotify with mobile module
+defaultModules.set(PNotifyMobile, {});
+
+// Create a notification stack
+const notifyStack = new Stack({
+  dir1: "down",
+  dir2: "left",
+  firstpos1: 25,
+  firstpos2: 25,
+  modal: false,
+  maxOpen: 3,
+  maxStrategy: "close",
+  maxClosureCausesWait: false,
+  push: "top",
+});
+
+// Function to create and show a notification
```

Comment:
Few suggestions.
- import React, {useSate} from React

Write the entire of this change as pure react. cc @mozzy11

Reply:
My whole point is . lets try to include react in our PRs.